### PR TITLE
support debian10 buster

### PR DESCRIPTION
**Pull request by pitan**
I want to use octopass in debian10.
This P/R is changed to build for debian 10 package.

### CODE REVIEW

**Comment by linyows** [comment]
Thank you for buster support.

File: octopass.c [hunk 1]
```diff
@@ -439,7 +439,7 @@ void octopass_github_request(struct config *con, char *url, struct response *res
 
   char *base = curl_escape(url, strlen(url));
   char dpath[MAXBUF];
-  char fpath[MAXBUF];
+  char fpath[MAXBUF+2];
```
Comment (linyows) [inline octopass.c]: What does this change mean?

Comment (hinaloe) [reply octopass.c]: I saw it...

> ```
> octopass.c: In function 'octopass_github_request':
> octopass.c:444:24: error: '-' directive writing 1 byte into a region of size between 0 and 1023 [-Werror=format-overflow=]
>   444 |   sprintf(fpath, "%s/%s-%s", dpath, base, octopass_truncate(con->token, 6));
>       |                        ^
> octopass.c:444:3: note: 'sprintf' output 3 or more bytes (assuming 1026) into a destination of size 1024
>   444 |   sprintf(fpath, "%s/%s-%s", dpath, base, octopass_truncate(con->token, 6));
>       |   ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
> cc1: all warnings being treated as errors
> make: *** [Makefile:52: nss_octopass] Error 1
> ==> ERROR: A failure occurred in build().
>     Aborting...
> ```

**Comment by linyows** [comment]
Sorry for the late merging!

**Comment by linyows** [comment]
Many thanks.

**Pull request merged**